iroha_client/tests/multisignature_transaction.rs

```diff
@@ -53,7 +53,9 @@ mod tests {
                 set_signature_condition.into(),
             ])
             .expect("Failed to prepare state.");
+        iroha_error::error!("Submit 1");
```

Comment:
These are definitely debug :) Please remove all `Submit 1`, `Submit 2` and `Submit 3` in this test.